### PR TITLE
Initial work on Hugo Theme

### LINKED ISSUE #250: Hugo theme
**Summary:** 

Design Hugo theme that has a similar look and feel as the Console.

**Why do we need this?**

Consistency in design.

**What is already there? What do you see now?**

Nothing.

**What is missing? What do you want to see?**

A Hugo theme.

Minimize time spent; focus on first visible things like fonts and colors.

**How do you propose to implement this?**

See [Create a Theme](https://gohugo.io/themes/creating/) and [Theme components](https://gohugo.io/themes/theme-components/).

**What can you do yourself and what do you need help with?**

Review.

### PR DESCRIPTION
**Pull request by htdvisser**
#### Summary

This Pull Request adds the initial Hugo theme for The Things Stack.

Closes #250

#### Changes

- Based on my previous work on the Hugo theme
- Merged in content and images from @SebastienCHEVALIER's #1425 
	- I made some CSS changes to make it look roughly the same, will fine-tune later
- Updated tooling
- Some small changes in content pages

#### Notes for Reviewers

I rendered on e31f510 https://htdvisser.github.io/lorawan-stack/feature-250-hugo-theme/ for review. 

Some of the summaries rendered on the overview pages are still missing or contain some text that shouldn't be there. We should update the markdown content to make this render better, but I'm not sure if that is in-scope for this PR.

### CODE REVIEW

**Comment by johanstokking** [comment]
Awesome. Looks great

**Comment by htdvisser** [comment]
Also moved some existing documentation to Hugo.

File: .mage/docs.go [hunk 1]
```diff
@@ -23,18 +24,41 @@ import (
...
 
 func execHugo(args ...string) error {
-	return execGo("run", append([]string{"github.com/gohugoio/hugo", "-s", "./doc"}, args...)...)
+	return execGo("run", append([]string{"-tags", "extended", "github.com/gohugoio/hugo", "-s", "./doc"}, args...)...)
 }
 
-// Deps installs documentation dependencies.
-func (Docs) Deps() error {
-	return sh.RunV("git", "submodule", "update", "--init", "doc/themes/hugo-theme-techdoc")
+func (d Docs) yarn() (func(args ...string) error, error) {
```
Comment (rvolosatovs) [inline .mage/docs.go]: We probably already have this in JS namespace, right? Why not just reuse that one?

Comment (htdvisser) [reply .mage/docs.go]: We actually have 2, one for JS and one for JS SDK because we're executing yarn from a different workdir.

Comment (rvolosatovs) [reply .mage/docs.go]: Can't we just have a `func yarn(wd string) (func(args ...string) error, error)`?

**Pull request merged**